frontend/views/containers/contributions/GroupPledgesGraph.vue

```diff
@@ -4,7 +4,7 @@
     :slices='mainSlices'
     :inner-slices='innerSlices'
   )
-    i18n.is-title-1.c-title(tag='h3' :args='LTags("span")') {span_}Group{_span} goal
+    i18n.has-text-1.is-title-3.c-title(tag='h3' :args='LTags("span")') {span_}Group{_span} goal
```

Comment:
We don't need `.is-title-3` here, this doesn't need to look like a title, is just a "normal" test with `.has-text-1` color.

Reply:
Mmm we actually stipulate in the guide that we need to tell the class all the time isn'
t it?

Reply:
We need to add the class, if the UI matches. in this case, the UI doesn't look like a title, so there's no need for it:
![image](https://user-images.githubusercontent.com/14869087/74129676-183ea800-4bd8-11ea-909e-f221c92184d5.png)

Reply:
Confusing.. anyway, there is no class is-text-[x] for 16px or 1rem

Reply:
A 16px h3 title with no class

Reply:
Okay, so another approach, that makes more sense:
Leave this "Group Goal" as a normal text (without heading) as we usually do.
Add the title of this graphic/summary top of the component and hide it using `.sr-only`

```pug
i18n.sr-only(tag='h3') Group Pledging Status
```

That way we serve both worlds visually and semantically.